episodes/22-external.md

```diff
@@ -26,8 +20,40 @@ exercises: 25
 
 ::::::::::::::::::::::::::::::::::::::::::::::::::
 
-::::::::::::::::::::::::::::::::::::::  challenge
+## Documenting your lesson
```

Comment:
This is the first of two episodes on collaboration skills: a pretty dramatic change of focus from the previous section, reflecting on teaching the lesson. I would like to include some introductory text about collaboration skills, before jumping right into the episode content. Maybe something framing these two episodes in the context of the training as a whole, and explaining how collaboration can benefit the development and maintenance of the lesson. In particular, for this episode I think it would be good to include a couple sentences explaining why it is worth taking time to make a project welcoming to newcomers.

Happy to help write some of this if you would like me to. Perhaps better kept to a separate PR?

Reply:
Sounds like a good idea @tobyhodges. I think I have now addressed every other comment on this PR and unless I draft an intro before you do another review - I will do it as a separate PR which may be easier to review/modify/discuss on.

Reply:
Done in 6469b75.